bench.sh

```diff
@@ -429,8 +429,17 @@ CABAL_BUILD_FLAGS=""
 
 GHC_VERSION=$(ghc --numeric-version)
 
+CABAL_BUILD_DIR="dist-newstyle"
+
 cabal_which() {
-  find dist-newstyle -type f -path "*${GHC_VERSION}*/$1"
+  if test -z "$CABAL_BUILD_DIR"
+  then
+    CABAL_BUILD_DIR="dist-newstyle"
+  fi
+  if test -d "$CABAL_BUILD_DIR/build"
```

Comment:
Why do you need this test? find would anyway fail if the directory does not exist.

Reply:
Yeah, I did not like the failure message:`find: ‘dist-newstyle/branch/build’: No such file or directory`. It gives the idea that something has gone wrong when it is completely normal.

Reply:
But then again, I'll be changing a few things so this might be removed. The problem was chart related.